### ISSUE TITLE
[Accessibility] [Password page] [AxeDev Tools] Decorative image is identified by screen reader and has no alt atribute

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#868]
**Describe the bug**
The decorative image of a man holding a key has no alt attribute. As a decorative image, it should be coded in CSS, not to be read by screen readers. Decorative images aren't important to be verbalized and may confuse navigation.
It's just identified an image and the user doesn't know what the image is or what is for.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to ['Muda's Password page'](https://muda.cambiatus.io/login?redirect=%2Fdashboard)
2. Run NVDA
3. Navigate until the element and see the error

**Expected behaviour**
As a decorative image, it should be in CSS to not be identified by screen readers. The screen reader user doesn’t know what is that image's purpose.

**Screenshots**
![image](https://github.com/user-attachments/assets/8a66f894-dc9d-46f3-9a73-62af564b37b0)

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS:Windows
- Browser Chrome
- Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
